openehr-rm/src/main/java/com/nedap/archie/rm/archetyped/Locatable.java

```diff
@@ -46,6 +42,25 @@
     @Nullable
     private List<Link> links = new ArrayList<>();
 
+
+    public Locatable() {
```

Comment:
As a very common superclass,  I think this needs both an equals and hashcode implementation so they super.equals/hashcode calls work as intended